include/zephyr/bluetooth/mesh/brg_cfg_srv.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ *  @brief Bluetooth Mesh Bridge Configuration Server Model APIs.
+ */
+#ifndef BT_MESH_BRG_CFG_SRV_H__
+#define BT_MESH_BRG_CFG_SRV_H__
+
+#include <zephyr/bluetooth/mesh.h>
+#include <zephyr/bluetooth/mesh/brg_cfg.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg_cli Bridge Configuration Server Model
+ * @ingroup bt_mesh
+ * @{
+ *  @brief API for the Bluetooth Mesh Bridge Configuration Server model
```

Comment:
```suggestion
 *  @brief API for the Bluetooth Mesh Subnetwork Bridge Configuration Server model
```

Reply:
Will keep the name from the Mesh Protocol spec.